src/Process/Builder/SubscriberBuilder.php

```diff
@@ -208,6 +198,7 @@ private function getInitialTestsConsoleLoggerSubscriber(TestFrameworkAdapter $te
 
     private function shouldSkipProgressBars(): bool
     {
+        // This really does not belong here. How's that going to be tested?..
         return $this->noProgress
             || getenv('CI') === 'true'
             || getenv('CONTINUOUS_INTEGRATION') === 'true';
```

Comment:
Need to be dealt with once the CI resolver from the replacement to #1067 comes into play.